src/main/java/io/kamax/matrix/client/_MatrixClientRaw.java

```diff
@@ -33,4 +35,8 @@
 
     _MatrixID getUser();
 
+    Optional<String> getDeviceId();
+
+    void logout();
```

Comment:
Not sure this belongs in _MatrixClientRaw... _MatrixClient might be better. I wish I added javadoc! I remember I split for a reason, but not the reason... let's discuss...